widdershins/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG NODE_VERSION=15.14-alpine
+
+FROM node:${NODE_VERSION}
```

Comment:
https://circleci.com/docs/ja/2.0/circleci-images/#nodejs
If you are creating an image to run on CircleCI, it is safer to base it on this image.